pymatgen/ext/tests/test_matproj.py

```diff
@@ -329,15 +329,15 @@ def test_get_pourbaix_entries(self):
             self.assertTrue(isinstance(pbx_entry, PourbaixEntry))
 
         fe_two_plus = [e for e in pbx_entries if e.entry_id == "ion-0"][0]
-        self.assertAlmostEqual(fe_two_plus.energy, -1.6228450214319294, places=2)
+        self.assertAlmostEqual(fe_two_plus.energy, -1.12369, places=3)
 
         feo2 = [e for e in pbx_entries if e.entry_id == "mp-25332"][0]
-        self.assertAlmostEqual(feo2.energy, 2.5523680849999995, places=2)
+        self.assertAlmostEqual(feo2.energy, 3.56356, places=3)
 
         # Test S, which has Na in reference solids
         pbx_entries = self.rester.get_pourbaix_entries(["S"])
         so4_two_minus = pbx_entries[9]
-        self.assertAlmostEqual(so4_two_minus.energy, 0.0644980568750011, places=2)
+        self.assertAlmostEqual(so4_two_minus.energy, 0.301511, places=3)
```

Comment:
@rkingsbury before merging, can you share verification that the new Pourbaix numbers are decent? have you compared the resulting Pourbaix diagrams? These differences seem like they may be significant.

Reply:
Hi @mkhorton thanks for flagging this; I should have said something about it. I saw it too and had the same reaction, but I checked many Pourbaix diagrams with the correction values in this MPR and they compare well with the existing ones. I'll email you a slide deck so you can see the comparisons. Bottom line I don't think we need to be concerned, and being able to do a full DB build will give us even more confidence.